src/dndbeyond/base/extras.js

```diff
@@ -64,24 +64,33 @@ class MonsterExtras extends CharacterBase {
         const attributes = stat_block.find("div[class*='styles_attribute__']");
         for (let attr of attributes.toArray()) {
             const label = $(attr).find("h2[class*='styles_attributeLabel']").text().trim();
-            const values = $(attr).find("p[class*='styles_attributeValue']");
+            const data = $(attr).find("p");
```

Comment:
You can do `$(attr).find("p").eq(0)` so you don't need to do `$(data[0])` afterwards (unless you need a potential array response for the `addAttribute` which uses the entire `data` object)